pkg/python/parse.go

```diff
@@ -34,8 +36,8 @@ func Parse(r io.Reader) ([]types.Library, error) {
 	return libs, nil
 }
 
-func stripComments(line *string) {
-	if pos := strings.IndexAny(*line, commentRune); pos >= 0 {
+func RstripByKey(line *string, key string) {
```

Comment:
Why do you need to export this function? And I prefer returning updated text like `strings.TrimPrefix`.

Reply:
Sorry, it was a mistake